src/app/config/config.c

```diff
@@ -8180,7 +8191,7 @@ getinfo_helper_config(control_connection_t *conn,
     for (i = 0; option_vars_[i].member.name; ++i) {
       const config_var_t *var = &option_vars_[i];
       /* don't tell controller about triple-underscore options */
```

Comment:
This comment is now incorrect, it should say "invisible options"?

Reply:
Yes; fixed in 65511a45adfeff